src/app/state/account/saga.ts

```diff
@@ -47,12 +48,33 @@ export function* fetchAccount(action: PayloadAction<string>) {
     ),
     join(
       yield* fork(function* () {
+        const networkType = yield* select(selectSelectedNetwork)
+
         try {
-          const transactions = yield* call(getTransactionsList, {
+          const transactions: Transaction[] = yield* call(getTransactionsList, {
             accountId: address,
             limit: TRANSACTIONS_LIMIT,
           })
-          yield* put(accountActions.transactionsLoaded(transactions))
+
+          const detailedTransactions = yield* call(() =>
+            Promise.allSettled(transactions.map(({ hash }) => getTransaction({ hash }))),
```

Comment:
This tries to incorrectly fetch paratime transactions too. See requests in:
https://wallet.dev.oasis.io/account/oasis1qry5d5zxs58w9deu5ara5zaamhkzqtg99s00ls90

Reply:
Right. Is there any better way to determine if the tx is a runtime tx -> https://github.com/oasisprotocol/oasis-wallet-web/pull/1999?